iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/SendMessagesIT.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.android;
+
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.MediumTest;
+
+import com.microsoft.azure.sdk.iot.common.iothubservices.SendMessagesCommon;
+import com.microsoft.azure.sdk.iot.device.DeviceClient;
+import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
+import com.microsoft.azure.sdk.iot.device.Message;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+@MediumTest
+@RunWith(AndroidJUnit4.class)
+public class SendMessagesIT {
+
+    public SendMessagesIT() {
+    }
+
+    //How much messages each device will send to the hub for each connection.
+    private static final Integer NUM_MESSAGES_PER_CONNECTION = 10;
+
+    // How much to wait until a message makes it to the server, in milliseconds
+    private static final Integer SEND_TIMEOUT_MILLISECONDS = 60000;
+
+    //How many milliseconds between retry
+    private static final Integer RETRY_MILLISECONDS = 100;
+
+    //Device Connection String
+    private String connString = "";
+
```

Comment:
I understand that service client does not work currently, but how are you fetching environmental variables for these Android tests. I don't see any code for that.

Reply:
Please add relevant code in setup and teardown excluding the use of service client if that is an issue.